prijateli_tree/app/routers/games.py

```diff
@@ -198,6 +206,22 @@ def get_previous_answers(
     }
 
 
+def language_check(player_lang, languages):
+    """
+    Helper function to ensure that the language is in the languages dict
+    """
+    if player_lang not in languages:
+        player_lang = player_lang.lower()
+
+    if player_lang not in languages:
+        raise HTTPException(
+            status_code=HTTPStatus.NOT_FOUND,
+            detail=f"language {player_lang} not found",
+        )
+
+    return player_lang
+
+
```

Comment:
I think it's worth wrapping the whole template_texr process in a this function and call it get_tempate_text or something. We'll do this over and over again.